weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsPage.tsx

```diff
@@ -47,68 +75,238 @@ export const CallsPage: FC<{
   paginationModel: GridPaginationModel;
   setPaginationModel: (newModel: GridPaginationModel) => void;
 }> = props => {
+  const {entity, project, baseView, views, fetchViews} = props;
   const [filter, setFilter] = useControllableState(
     props.initialFilter ?? {},
     props.onFilterUpdate
   );
 
   const isEvaluationTable = useCurrentFilterIsEvaluationsFilter(
     filter,
-    props.entity,
-    props.project
+    entity,
+    project
   );
+  // table is the internal id stored in the object.
+  const table = isEvaluationTable ? 'evaluations' : 'traces';
+
+  const getTsClient = useGetTraceServerClientContext();
+
+  const tsClient = getTsClient();
+  const view = baseView.object_id;
 
-  const title = useMemo(() => {
-    if (isEvaluationTable) {
-      return 'Evaluations';
+  const currentViewDefinition = useCurrentViewDefinition(baseView);
+
+  const history = useHistory();
```

Comment:
it is always a bit of a smell to me when a component uses history below the main routing component (in our case Browse3). It means that CallsPage itself is now strictly dependent on the URL and cannot be embedded elsewhere. This particular case seems challenging to separate, so maybe we can punt on this comment. However, we should acknowledge the impact of this change